src/schema.cpp

```diff
@@ -77,6 +78,70 @@ Schema::const_iterator Schema::find(ObjectSchema const& object) const noexcept
     return const_cast<Schema *>(this)->find(object);
 }
 
+namespace {
+
+struct CheckObjectPath
+{
```

Comment:
```suggestion
struct CheckObjectPath {
```

Only functions have a newline before {